packages/scoutgame/src/tokens/divideTokensBetweenDeveloperAndHolders.ts

```diff
@@ -85,49 +78,49 @@ export function divideTokensBetweenBuilderAndHolders({
     const scoutReward = calculateRewardForScout({
       purchased: { default: owner.totalNft, starterPack: owner.totalStarter },
       supply: { default: nftSupply, starterPack: starterPackSupply },
-      scoutsRewardPool: earnableScoutTokens
+      scoutsRewardPool: earnableTokens
     });
     const scoutTokens = Math.floor(scoutReward);
 
     return { scoutId: owner.scoutId, nftTokens: owner.totalNft, erc20Tokens: scoutTokens };
   });
 
-  const tokensForBuilder = Math.floor((defaultBuilderPool * earnableScoutTokens) / 100);
+  const tokensForDeveloper = Math.floor((defaultDeveloperPool * earnableTokens) / 100);
```

Comment:
all the division done should probably be done with BigInts. This removes the need to use Math.floor, and allows more accuracy